### PR TITLE
Blog implementation

### PR DESCRIPTION
**Pull request by stefonarch**
Fixes https://github.com/lxqt/lxqt.github.io/issues/92

* promoted: true|false : display or not on main page
* timestamps need to be be changed before merging
* edit/improve postings, specially the qt6 migration?

schermata-02-10-14-15

### CODE REVIEW

**Comment by tsujan** [comment]
It isn't easy to keep track of git changes. But if these previews can be updated, why not?

**Comment by stefonarch** [comment]
It's non meant to be updated, we can just write a new post about whatever, it hasn't to be frequent, but it gives a message of being alive, most people don't follow GH. 
Example: https://blog.xfce.org/

**Comment by stefonarch** [comment]
Any improvements/corrections for the first 2 postings?

File: _posts/2022-01-11-lxqt-1.1-preview.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+
+<br/>
+<img src="../../../../../images/posts/plasma-logo-dark.png" width="600"/>
+
+
+<br/>
+### xdg-desktop-portal-lxqt
+
+The backend implementation of [xdg-desktop portal](https://github.com/flatpak/xdg-desktop-portal#readme) is ready for releasing! It allows to use the lxqt filedialog in GTK and other applications. See the [wiki](https://github.com/lxqt/lxqt/wiki/ConfigMiscellaneous#lxqt-filedialog-in-gtk-applications) for details.
```
Comment (palinek) [inline _posts/2022-01-11-lxqt-1.1-preview.md:68]: I would explicitly say somewhere, that it is only partial implementation of the xdg-desktop-portal. As we really implement only the FileChooser API and non of the others APIs.

Comment (stefonarch) [reply _posts/2022-01-11-lxqt-1.1-preview.md:68]: Could you write this text? You're way more into it.

Comment (palinek) [reply _posts/2022-01-11-lxqt-1.1-preview.md:68]: `Backend implementation of [xdg-desktop-portal](https://github.com/flatpak/xdg-desktop-portal#readme) (currently only [org.freedesktop.portal.FileChooser](https://flatpak.github.io/xdg-desktop-portal/#gdbus-org.freedesktop.portal.FileChooser) API) is ready for releasing!`

Comment (stefonarch) [reply _posts/2022-01-11-lxqt-1.1-preview.md]: Thanks!

**Pull request merged**